common/api-review/util.api.md

```diff
@@ -193,6 +193,8 @@ export interface FirebaseDefaults {
     config?: Record<string, string>;
     // (undocumented)
     emulatorHosts?: Record<string, string>;
+    // (undocumented)
```

Comment:
This should probably be documented, I will leave a comment below where you do this.